DigitalLearningSolutions.Web/Controllers/TrackingSystem/Centre/ContractDetails/ContractDetailsController.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace DigitalLearningSolutions.Web.Controllers.TrackingSystem.Centre.ContractDetails

```

Comment:
This probably doesn't need to be in it's own folder.. If anything I'd probably group all the dashboard-related pages into a "Dashboard" folder, but tbh, I think the controller being in the "Centre" folder is sufficient